### PR TITLE
[15.0] Migration of base_comment_template

### PR DESCRIPTION
**Pull request by alan196**
### Proposed changes

Migration PR for module base_comment_template.

### Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Migration Update

### Checklist

- [x] I have read the CONTRIBUTING doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the necessary documentation

### CODE REVIEW

**Review by JuanyDForgeflow — APPROVED** [review]
LGTM

**Comment by marylla** [comment]
@alan196 Hi there, could you please rebase? I would like to test it on runboat.

**Comment by rousseldenis** [comment]
@alan196

**Comment by alan196** [comment]
@marylla @rousseldenis rebased 👍 sorry I was on holidays

**Comment by victoralmau** [comment]
I see many changes from v14 that are not in the history (specifically since May 26), can you add them?

**Comment by alan196** [comment]
@victoralmau updated with latest changes of V14 👍

File: base_comment_template/models/comment_template.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                    ("model_ids.model", "=", self._name),
+                ]
+            )
+            for template in templates:
+                domain = safe_eval(template.domain)
+                if not domain or record.filtered_domain(domain):
+                    record.comment_template_ids = [(4, template.id)]
+
+    def render_comment(
+        self, comment, engine="jinja", add_context=None, post_process=False
```
Comment (victoralmau) [inline base_comment_template/models/comment_template.py:47]:
```suggestion
        self, comment, engine="jinja", add_context=None, post_process=False
```

According to the documentation, `jinja` is not allowed in this function: https://github.com/odoo/odoo/blob/15.0/addons/mail/models/mail_render_mixin.py#L489

**Review by victoralmau — CHANGES_REQUESTED** [review]
Can you resolve pre-commit and test errors?

**Comment by victoralmau** [comment]
Superseed by: https://github.com/OCA/reporting-engine/pull/646

**Pull request closed without merging**